### PR TITLE
Rs 2022 03 01

### PR DESCRIPTION
**Pull request by rschlaikjer**
Getting into the fat files again :sacchincopium:

### CODE REVIEW

File: script/Arcueid/Day 8/08_00_ARC08_6.txt [hunk 1]
```diff
@@ -36,27 +37,31 @@ But for whatever reason, Arcueid shouts at me as soon as she sees my face.
 [sha:87aa5ed28f5d574621f426ea5f514377fb345fa7]{
 -- Page 290, Offset 20396.
 -- 「そう！　もう、何時だと思ってるのよ！
-"That's right! Geez, what time did you think it was?
+"That's right! Geez, what time do you call this?
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_6.txt:40]: revert

File: script/Arcueid/Day 8/08_00_ARC08_6.txt [hunk 2]
```diff
@@ -131,33 +141,36 @@ I mean, in the first place, what was she planning to do, coming here three hours
...
 -- 「……それはそうだけど。
-"Well that's true, but...
+"Well, that's true, but...
 }
 [sha:8595a8e0817de0661919124e29bb9c31c7821d71]{
 -- Page 302, Offset 20420.
 -- しっかし、なんだって３時間も待ってたんだ。間違えて来たにしろ、それだけあれば部屋に戻っていても良かったじゃないか」
-// why /did/ you wait for three hours? italics candidate
-I mean, why did you even wait for three hours? Even if you made a mistake, you could have just gone back to your room."
+// Are we translating Arc's 部屋 literally? 'Room' is a bit of a weird way to refer to what looks a lot like an apartment
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_6.txt:171]: Yes. 部屋 is "apartment", not room here.

I think we had this discussion before too in day 4

File: script/Arcueid/Day 8/08_00_ARC08_6.txt [hunk 3]
```diff
@@ -197,62 +214,74 @@ I think I'm going crazy, but I can't figure out what's wrong with my body at all
...
 -- 「ま、いいわ。
-"Well enough with that.
+"Well, never mind all that.
 }
 [sha:5e0b110b5b9cb94b18a4df9d6bac12b61d5acbaf]{
 -- Page 306, Offset 20431. Glued.
 -- 時間もなくなっちゃったし、無駄話をしている暇なんてないものね」
-We don't have much time, so we can't afford to be idly chatting around."
+// Leading space for glue
+ Time's a wasting, we can't just stand here shooting the breeze all night."
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_6.txt:235]: I think this is too much. Revert imo

File: script/Arcueid/Day 8/08_00_ARC08_6.txt [hunk 4]
```diff
@@ -352,32 +389,37 @@ We're going to be looking for vampires just like yesterday, right?"
...
-Arcueid said that she's headed toward a specific destination, but because I don't know the details, it looks as if the two of us are aimlessly walking around the night city.
+// What did I JUST say
+Arcueid said she had a specific destination in mind, but hasn't let me in on the details.
+ As a result, it feels more to me as though the two of us are simply taking a leisurely stroll together through the city streets.
 }
 [sha:05cdbe2f523ea3140f6984887ba602f96a824b77]{
 -- Page 318, Offset 20462.
 -- 「今夜はピンポイントで視てもらうわ。まずは騒がしい方ね」
-"This time I'll have you look at a pinpointed location. Let's begin with the noisy one."
+"Tonight I'll have you look at some locations I pinpointed. We'll start with the bustling one."
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_6.txt:411]: I think there needs to be something before bustling.
> Let's start with the most bustling one.

File: script/Arcueid/Day 8/08_00_ARC08_6.txt [hunk 5]
```diff
@@ -402,37 +444,43 @@ the white vampire whose form hasn't changed through the centuries.
...
+"So it's a miss. On to the next one then.
+ Right, put your glasses back on.
+ The next one's quite far, so don't push yourself too hard."
 }
 [sha:830dd6be740fa3e54669e2a5523791abffd917d6]{
 -- Page 322, Offset 20473.
 -- 昼間言っていたように、アルクェイドは死者たちの巡回場所に当たりを付けていた。
-Just as she said during the day, Arcueid has a good idea of the areas where the Dead are patrolling.
+// Scanned back through 08_00_ARC08_3_1 but I don't see her actually mention this explicitly. Oh well.
+Arcueid had mentioned this afternoon that she'd made some educated guesses about where the Dead might be patrolling.
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_6.txt:473]: I think this is just referencing whats implied by 20462

File: script/Arcueid/Day 8/08_00_ARC08_6.txt [hunk 6]
```diff
@@ -487,7 +540,8 @@ The time is close to midnight.
 [sha:b21b08e12fb4eaf0caaecedcae08092566006796]{
 -- Page 328, Offset 20486.
 -- さすがに歩行者の数は減り、ネオンの光も減少していく。
-As expected, the number of pedestrians decreased and the neon lights also diminished.
+As one would expect, there aren't as many pedestrians about,
+ and the lit neon lights are also becoming fewer and further between.
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_6.txt:544]: in American we say "fewer and farther between"

File: script/Arcueid/Day 8/08_00_ARC08_6.txt [hunk 7]
```diff
@@ -592,47 +658,50 @@ Hey. Are you really okay with being with me? Aren't you afraid of slaying vampir
...
 -- Page 339, Offset 20513.
 -- 会話はそこで途切れた。
-From there, the conversation comes to an end.
+With that, our conversation comes to a halt.
 }
 [sha:cf3c2f850d1ce6c24fac44039efc6ac5841dd27b]{
 -- Page 339, Offset 20514.
 -- 俺たちは無言で、すれ違う者のいない歩道を歩いていく。
-In complete silence, we walk down the sidewalk with no one passing by.
+In complete silence, we walk down deserted sidewalk.
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_6.txt:698]: the deserted sidewalk.

File: script/Arcueid/Day 8/08_00_ARC08_6.txt [hunk 8]
```diff
@@ -592,47 +658,50 @@ Hey. Are you really okay with being with me? Aren't you afraid of slaying vampir
...
 }
 [sha:46b861f2b25b7b5b18d5537d6c630f70b774b3ed]{
 -- Page 340, Offset 20515.
 -- 駅前から随分と離れた。この２時間、成果はほぼ無かったが不思議と徒労感はない。
-We've come a long way from the station. The past two hours have been almost completely lacking in results, but strangely enough, it didn't feel like a waste.
+We've come a long way from the station.
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_6.txt:703]: This makes it sound almost like they've made great progress emotionally. I'd propose just "We're a long way from the station"

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 8/08_00_ARC08_6.txt (hunk 3 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 8/08_00_ARC08_6.txt:235]: Agree that time's a wasting is pushing it, but can we keep shooting the breeze? "idly chatting around" doesn't really parse to me

Comment (Avuxo) [reply script/Arcueid/Day 8/08_00_ARC08_6.txt:235]: yeah, that parts ok

**Pull request merged**